experimental/FTheoryTools/src/AbstractFTheoryModels/precomputed_attributes.jl

```diff
@@ -0,0 +1,340 @@
+##########################################
+### (1) Meta data getters
+##########################################
+
+meta_data = [
+    (:associated_literature_models, Vector{String}),
+    (:arxiv_doi, String),
+    (:arxiv_id, String),
+    (:arxiv_link, String),
+    (:arxiv_model_equation_number, String),
+    (:arxiv_model_page, String),
+    (:arxiv_model_section, String),
+    (:arxiv_version, String),
+    (:birational_literature_models, Vector{String}),
+    (:journal_doi, String),
+    (:journal_link, String),
+    (:journal_model_equation_number, String),
+    (:journal_model_page, String),
+    (:journal_model_section, String),
+    (:journal_pages, String),
+    (:journal_report_numbers, Vector{String}),
+    (:journal_volume, String),
+    (:journal_name, String),
+    (:journal_year, String),
+    (:literature_identifier, String),
+    (:model_description, String),
+    (:model_parameters, Dict{String, Int64}),
+    (:paper_authors, Vector{String}),
+    (:paper_buzzwords, Vector{String}),
+    (:paper_description, String),
+    (:paper_title, String)
+]
+
+for (name, typ) in meta_data
+    @eval @define_model_attribute_getter ($name, $typ)
+end
```

Comment:
This eval for loop doesn't save anything here.
Please convert it to
```julia
@define_model_attribute_getter(:associated_literature_models, Vector{String})
@define_model_attribute_getter(:arxiv_doi, String)
```
etc. to avoid another unnecessary `@eval`

Reply:
I was thinking that it would be nice to have such a list eventually. As we might loop over this or similar lists of functions in more places. But for now, I just remove the list and coded these instances as you suggested.